ts-packages/web/src/app/(social)/_components/News.tsx

```
@@ -41,7 +41,9 @@ export default function News() {
             </h4>
             <p
               className="text-sm/[20px] align-middle font-light line-clamp-2 whitespace-normal"
-              dangerouslySetInnerHTML={{ __html: item.html_content }}
+              dangerouslySetInnerHTML={{
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)